content/concepts/asset-manager.md

```diff
@@ -601,10 +727,6 @@ In fine, below is the JSON of our dear `XMLD500_fr_FR_user_guide` asset, once we
 }
 ```
 
-::: info
-Those two new attributes can be easily filled by using the API and a simple regular expression on the code of the asset, as both information, the product reference and the locale are already in the code.
-:::
-
 So, now that our asset is ready, whenever the rule is launched, the PIM will automatically extrapolate the rule, by replacing the curlies reference, `{{product_ref}}` and `{{locale}}`, by their real values in the asset it's currently trying to link to products.
```

Comment:
```suggestion
So, now that our asset is ready, whenever the rule is launched, the PIM will automatically extrapolate the rule. It will replace the curlies reference, `{{product_ref}}` and `{{locale}}` by their actual values in the asset it's currently trying to link to products.
```